### DISCUSSION TITLE
Sending ^V/lnext to the shell

### DISCUSSION THREAD

**Discussion by dumain** [kovidgoyal/kitty#5003] [General]
I like all my apps to use the same keys for copy and paste.  Since most GUI apps use CTRL-C and CTRL-V for copy and paste I want kitty to do the same.  This is easy enough to set up in kitty however I would also like kitty to send the
lnext character(CTRL-V) to the shell when I type CTRL-SHIFT-V.  i've tried `map ctrl+shift+v send_text \x16  ` and` map ctrl+shift+v no_op` but pressing CTRL-SHIFT-V produces no effect.  any clues?  This works more or less automagically with qterminal but it is otherwise not as powerful as kitty.  currently using kitty 0.20.3 from guix

**Comment by kovidgoyal** [comment]
You need to use the correct syntax for send_text. See the docs for it.

**Reply by dumain** [reply]
Thanks I now have: map `ctrl+shift+v send_text all \x16` and it works.
